model/cloud.go

```diff
@@ -312,11 +312,16 @@ type CreateSubscriptionRequest struct {
 	DiscountID            string   `json:"discount_id"`
 }
 
-type DowngradeFeedback struct {
+type Feedback struct {
 	Reason   string `json:"reason"`
 	Comments string `json:"comments"`
 }
 
+type WorkspaceDeletionRequest struct {
+	SubscriptionID string `json:"subscription_id"`
```

Comment:
Will there always be a subscription ID?

Cloud Free is allowed to delete their workspaces, will they have a subscription?
(1/5) I guess there would likely be one in stripe, so I think it makes sense to always expect one.

Reply:
Yup, all cloud free workspaces have backing Stripe subscriptions.